fecgov/openFEC
# add presidential tables for 2008 and 2012 cycles

## Summary (required)

- Resolves #5447 

(Include a summary of proposed changes and connect issue below)

### Required reviewers

This PR requires a reviewer from database team.

## Impacted areas of the application

None

-  

## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)

## Related PRs

Related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()

## How to test
- download this branch to local server. Run flyway migrate (or create_sample_db). 
- Make sure migration finish successfully.
- log into cfdm_test, make sure the following 18 tables created:
disclosure.pres_ca_cm_sched_a_join_08d
disclosure.pres_nml_ca_cm_link_08d
disclosure.pres_ca_cm_sched_state_08d
disclosure.pres_ca_cm_sched_link_sum_08d
disclosure.pres_nml_sched_a_08d
disclosure.pres_nml_sched_b_08d
disclosure.pres_f3p_totals_ca_cm_link_08d
disclosure.pres_nml_f3p_totals_08d
disclosure.pres_nml_form_3p_08d
disclosure.pres_ca_cm_sched_a_join_12d
disclosure.pres_nml_ca_cm_link_12d
disclosure.pres_ca_cm_sched_state_12d
disclosure.pres_ca_cm_sched_link_sum_12d
disclosure.pres_nml_sched_a_12d
disclosure.pres_nml_sched_b_12d
disclosure.pres_f3p_totals_ca_cm_link_12d
disclosure.pres_nml_f3p_totals_12d
disclosure.pres_nml_form_3p_12d

- review the ddl for the following 5 views, make sure it include the 2008 and 2012 presidential tables:
public.ofec_presidential_coverage_date_vw
public.ofec_presidential_by_candidate_vw
public.ofec_presidential_by_state_vw
public.ofec_presidential_financial_summary_vw
public.ofec_presidential_by_size_vw

 

## System architecture updates (if applicable)

(If this pull request changes our [current system diagram](https://github.com/fecgov/FEC/wiki/2.-FEC-system-diagram), include a description of those changes here and create a new ticket to update the system diagram)
